README.md

```diff
@@ -190,8 +190,9 @@ Sanitizer                              | Description
 **toInt(input [, radix])**             | convert the input string to an integer, or `NaN` if the input is not an integer.
 **trim(input [, chars])**              | trim characters (whitespace by default) from both sides of the input.
 **unescape(input)**                    | replaces HTML encoded entities with `<`, `>`, `&`, `'`, `"` and `/`.
-**whitelist(input, chars)**            | remove characters that do not appear in the whitelist. The characters are used in a RegExp and so you will need to escape some chars, e.g. `whitelist(input, '\\[\\]')`.
-
+**whitelist(input, chars)**            | remove characters that do not appear in the whitelist. The characters are used in a
```

Comment:
Please undo this change, the text should remain on 1 line so the table is being parsed correctly